locales/core/de.json

```diff
@@ -0,0 +1,207 @@
+{
+  "account": "Account",
```

Comment:
Was this translated?

Reply:
I always "Account" because "Konto" sounds awful. But I can change it if you want.

Reply:
No, I'm just not a native German speaker and like I mentioned Google just said it wasn't translated. I trust you as a native speaker, so leave it the way it is